src/services/tripsService.js

```diff
@@ -83,6 +86,29 @@ class tripHelper {
 
     return allTrips;
   }
+
+  /**
+   * @param {string} page
+   * @param {string} limit
+   * @returns {object} most traveled centres infos
+   */
+  static async getTrendingCentres(page, limit) {
+    const pagination = paginate(page, limit),
+      mostTraveledCentres = await Booking.findAll({
+        group: ['accommodationId', 'Accommodation.id'],
+        attributes: ['accommodationId', [Sequelize.fn('COUNT', 'accommodationId'), 'peopleVisited']],
+        include: [{
+          model: Accommodations,
+          attributes: {
+            exclude: ['createdBy', 'id', 'createdAt', 'updatedAt']
+          }
+        }],
+        order: [[Sequelize.col('peopleVisited'), 'DESC']],
+        offset: pagination.offset,
+        limit: pagination.limit,
+      });
+    return mostTraveledCentres;
+  }
```

Comment:
if this would be added to accommodationService would be fine too

Reply:
@blaise82 Thanks again for this, I've fixed it